pkg/catalogv2/oci/oci.go

```diff
@@ -121,7 +121,7 @@ func GenerateIndex(ociClient *Client, URL string, credentialSecret *corev1.Secre
 
 	var maxTag string
 	var chartName string
-
+	var updatedChartVersions []*repo.ChartVersion
```

Comment:
Can we also test the case where we have multiple repositories ? `docker.io/rancher/`